File: python/card.py
from ast import Num
from payment import Payment
class Card(Payment):
    carNumber = str
    carName = str
    cvv = int
    expireDate = str
    def __init__(self, id, carNumber, carName, cvv, expireDate) -> None:
        super().__init__(id)
        self.carNumber = carNumber
        self.carName = carName
        self.cvv = cvv
        self.expireDate = expireDate

File: php/card.php
<?php
    require('payment.php');
    class Card extends Payment {
        public $numberCard;
        public $nameCard;
        public $cvv;
        public $expireDate;
        public funtion __construct($id, $numberCard, $nameCard, $cvv, $expireDate) {
            parent::__construct($id);
            this->numberCard = $numberCard;
            this->nameCard = $nameCard;
            this->cvv = $cvv;
            this->expireDate = $expireDate;
        }
    }
?>